README.md

```diff
@@ -1 +1,93 @@
-# FIrst-Microverse
\ No newline at end of file
+ # [First-Microverse-Hello-Project]
```

Comment:
- [ ] As suggested by the previous reviewer, In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.
- The menu is a requirement
- The only optional section is the FAQ

| professional Readme  | Your Readme that is not professional |
| ------------- | ------------- |
|  ![Recording 2023-02-14 at 12 52 20 (5)](https://user-images.githubusercontent.com/83360636/218715072-2b2d3a49-8219-4939-b350-45cc950f7f7a.gif)	  | ![Untitled__2__AdobeExpress](https://user-images.githubusercontent.com/52934128/220406716-c0c1a4e8-e974-44e0-8000-b1081a37942b.gif)
      |